webapp/app/tools/admin_functions/views/admin_functions.html

```diff
@@ -1,65 +1,106 @@
+            <hr/>
+            <h3>Super Admin Functions</h3>
 
-    <div class="panel panel-default">
-        <div class="panel-heading">
-          <h4 class="panel-title">
-            <a ng-click="isRegionCollapsed = !isRegionCollapsed">
-              <label>Regions</label>
-            </a>
-          </h4>
-        </div>
-        <div collapse="isRegionCollapsed" id="regionsCollapsable" collapsed>
-          <div class="panel-body">
-              <form ng-submit="submitNewRegion()" id="newRegionForm">
-                  <h4>Add New Region</h4>
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h4 class="panel-title">
+                    <a ng-click="isRegionCollapsed = !isRegionCollapsed">
+                      <label>Regions<span class="caret"></span></label>
+                    </a>
+                  </h4>
+                </div>
+                <div collapse="isRegionCollapsed" id="regionsCollapsable" collapsed>
+                  <div class="panel-body">
+                      <form ng-submit="submitNewRegion()" id="newRegionForm">
+                          <h4>Add New Region</h4>
+
+                          <p ng-model="regionStatusMessage" id="regionStatusMessage"></p>
+                          <label>Name:</label> <input type="text" id="region_name_text" class="form-control mobile" ng-model="postParams.new_region"><br/>
+
+                          <input class="btn btn-success mobile_small" type="submit" id="submit_new_region_btn" value="submit">
+                      </form>
+                  </div>
+                </div>
+            </div>
+
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h4 class="panel-title">
+                    <a ng-click="isUserCollapsed = !isUserCollapsed">
+                      <label>Users<span class="caret"></span></label>
+                    </a>
+                  </h4>
+                </div>
+                <div collapse="isUserCollapsed" id="userCollapsable">
+                  <div class="panel-body">
+                      <form ng-submit="submitNewUser()" id="newUserForm">
+                        <h4>Add New Admin</h4>
+                        <p ng-model="userStatusMessage" id="userStatusMessage"></p>
+                        <label>Username:</label> <input type="text" id="new_user_name" class="form-control mobile" ng-model="postParams.new_user_name"> <br/>
+                        <label>Password:</label> <input type="password" id="new_user_pass" class="form-control" ng-model="postParams.new_user_pass"> <br/>
+                        <br/>
+
+                        <h4>Permission Level:</h4>
+                        <select ng-model="postParams.new_user_permissions" id="permissionLevelSelect">
+                            <option>Region Admin</option>
```

Comment:
can we put value='REGION' and value='SUPER' here to pass 'REGION' and 'SUPER' to the backend? (not a big deal, but it's nice when values are consistent between frontend and backend)